routing/record/record.go

```diff
@@ -0,0 +1,42 @@
+package record
```

Comment:
cool! i think `(dht *IpfsDHT) verifyRecord` also belongs in this package. I'll move it over